WcaOnRails/app/assets/stylesheets/teams-and-committees.scss

```diff
@@ -56,6 +62,9 @@
       vertical-align: top;
       border-radius: 5px;
     }
+    a {
```

Comment:
I think this scss file needs a bit of refactoring to DRY the common styles of all badges and taking advantage of SASS's `@extend` feature.
Definitely not for that PR, but if you feel up for it later it would be awesome!

Reply:
Noted, will do it after this as that file will probably end up much more cleaner